website/docs/reference/define-properties.md

```diff
@@ -40,7 +40,7 @@ These properties are:
 - [`source` properties](/reference/source-properties) (for example, `loaded_at_field`, `freshness`)
 - [`exposure` properties](/reference/exposure-properties) (for example, `type`, `maturity`)
   - Note that while most exposure properties must be configured directly in `properties.yml` files, you can set the [`enabled`](/reference/resource-configs/enabled) config at the [project level](/reference/exposure-properties#project-level-configs) in the`dbt_project.yml` file.
-- [`macro` properties](/reference/macro-properties) (for example, `arguments`)
+- [`macro` properties](/reference/resource-properties/arguments) (for example, `arguments`)
```

Comment:
I don't think we want to update this to point to the specific `argument`s page; instead it could point to the general "macro properties page". same note for the other link updates

Reply:
Agree! I think we mistakenly removed the macro props page.